Dockerfile

```
@@ -16,18 +15,52 @@ RUN apt-get -y update
 RUN apt-get -y install git wget xz-utils build-essential \
                        curl dejagnu subversion bison flex bc libcap-dev \
                        cmake libncurses5-dev libboost-all-dev \
-                       default-jdk sbt
+                       default-jdk sbt --no-install-recommends
 
 
 # Installing Angelix
 
-RUN git clone --recursive https://github.com/mechtaev/angelix.git
+RUN apt-get install software-properties-common -y --no-install-recommends
+
+RUN git clone --recursive https://github.com/mechtaev/angelix.git --depth 1
+
+
+RUN apt-get purge icedtea-* openjdk-* -y
+RUN sudo add-apt-repository -y ppa:openjdk-r/ppa; sudo apt-get update; sudo apt-get install -y openjdk-8-jdk
```

```suggestion
RUN sudo add-apt-repository -y ppa:openjdk-r/ppa && sudo apt-get update && apt-get install -y openjdk-8-jdk
```

Use `&&` is more appropriate.